Source/Utils/ParticleUtils.H

```diff
@@ -17,6 +17,75 @@
 
 namespace ParticleUtils {
 
+    /**
+     * \brief Determine which cell a particle lies in.
```

Comment:
Title+body notation is what I would prefer here, use like this:
```suggestion
    /**  Determine which cell a particle lies in.
     *
```